examples/monique/calibration/seasonality_calibration/01_burnin_for_seasonalityCalib/set_config.py

```diff
@@ -0,0 +1,58 @@
+import copy
+import emod_api.config.default_from_schema_no_validation as dfs
+from emodpy_malaria.malaria_config import configure_linear_spline, set_species_param
+from snt.calibration.helpers_seasonality_calibration import get_burnin_spline_values
+from snt.helpers_sim_setup import update_basic_params, set_input_files, set_drug_params
+
+import manifest
+import params
+
+
+def set_config(config):
+    update_basic_params(config, manifest, manifest.project_path)
+
+    set_input_files(config, params.rep_admin, params.rep_admin,
+                    population_size=params.simulation_pop)  # climate and demographics files
+
+    set_drug_params(config)
+
+    if params.serialize:
+        config.parameters.Simulation_Duration = params.years * 365 + 1
+        config.parameters.Serialization_Time_Steps = [365 * (params.years - 5), 365 * params.years]
+        config.parameters.Serialized_Population_Reading_Type = 'NONE'
+        config.parameters.Serialized_Population_Writing_Type = 'TIMESTEP'
+        config.parameters.Serialization_Mask_Node_Write = 0
+        config.parameters.Serialization_Precision = 'REDUCED'
+    else:
+        config.parameters.Serialized_Population_Reading_Type = 'NONE'
+        config.parameters.Simulation_Duration = params.years * 365 + 1

```

Comment:
We can remove "writing_type"/"reading_type" = None, because those are the defaults, but it doesn't hurt keeping them in to explicitly tell the user nothing is happening maybe?

Reply:
> We can remove "writing_type"/"reading_type" = None, because those are the defaults, but it doesn't hurt keeping them in to explicitly tell the user nothing is happening maybe?

[ZDU] Done, removed, thanks!